docs/content/en/docs/implementing/add-app-awareness/index.md

```diff
@@ -52,13 +52,12 @@ spec:
         component: podtato-head-right-leg
       annotations:
         keptn.sh/workload: "right-leg"
-        keptn.sh/version: "0.1.0"
```

Comment:
I think this was deleted by accident, can you put it back? 😸

Reply:
right, I'll just put that line back :)